homeassistant/components/tts/__init__.py

```diff
@@ -618,9 +618,9 @@ def write_tags(
                 if not tts_file.tags:
                     tts_file.add_tags()
                 if isinstance(tts_file.tags, ID3):
-                    tts_file["artist"] = ID3Text(encoding=3, text=artist)
-                    tts_file["album"] = ID3Text(encoding=3, text=album)
-                    tts_file["title"] = ID3Text(encoding=3, text=message)
+                    tts_file["artist"] = ID3Text(encoding=3, text=artist)  # type: ignore[no-untyped-call]
+                    tts_file["album"] = ID3Text(encoding=3, text=album)  # type: ignore[no-untyped-call]
+                    tts_file["title"] = ID3Text(encoding=3, text=message)  # type: ignore[no-untyped-call]
```

Comment:
mutagen expresses itself to PEP561 compatible by being typed, however, ID3Text() (which inherits from Fram) is not typed... 

<https://github.com/quodlibet/mutagen/blob/d98d92736c071811838d76a6ed3731b26089fc6f/mutagen/id3/_frames.py#L55>

For us there is no other option than ignoring it.